portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,4 +35,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String json = gson.toJson(comments);
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // get input from form
```

Comment:
[Nit] Comments are more readable when formatted like standard English (meaning they start with a capital letter and end with punctuation.

In this case I would change this to
"Get input from form."